File: app/src/main/java/net/radityalabs/contactapp/presentation/util/CollectionUtil.java
package net.radityalabs.contactapp.presentation.util;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by radityagumay on 4/12/17.
 */

public class CollectionUtil {

    public static <T> List<T> join(List<T>... lists) {
        List<T> result = new ArrayList<T>();
        for (List<T> list : lists) {
            result.addAll(list);
        }
        return result;
    }

    public static <T> boolean isValid(List<T> collections) {
        return collections != null && collections.size() > 0;
    }
}

File: app/src/main/java/net/radityalabs/contactapp/presentation/util/CollectionUtil.kt
package net.radityalabs.contactapp.presentation.util

import java.util.ArrayList

/**
 * Created by radityagumay on 4/12/17.
 */

object CollectionUtil {

    fun <T> join(vararg lists: List<T>): List<T> {
        val result = ArrayList<T>()
        for (list in lists) {
            result.addAll(list)
        }
        return result
    }

    fun <T> isValid(collections: List<T>?): Boolean {
        return collections != null && collections.size > 0
    }
}
